core/src/FundingSource/Eligibility/Checker/PaylaterEligibilityChecker.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License version 3.0
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License version 3.0
+ */
+
+namespace PsCheckout\Core\FundingSource\Eligibility\Checker;
+
+use PsCheckout\Core\Settings\Configuration\PayPalIntentConfiguration;
+
+class PaylaterEligibilityChecker extends BaseFundingSourceEligibilityChecker
+{
+    protected function getSupportedName(): string
+    {
+        return 'paylater';
+    }
+
+    protected function getRequiredCountryIsoCodes(): array
+    {
+        return ['FR', 'GB', 'US', 'ES', 'IT'];
```

Comment:
```suggestion
        return ['FR', 'GB', 'US', 'ES', 'IT', 'CA'];
```
PayPal added Canada recently https://developer.paypal.com/docs/checkout/pay-later/ca/

Reply:
I removed this property as it is a duplication of `FundingSourceConstraint::getCountries()` method. This will prevent any miss if we need to add new countries.